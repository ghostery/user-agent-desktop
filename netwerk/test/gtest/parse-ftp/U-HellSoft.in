ftp> open pandora.anglistik.uni-mainz.de
220 pandora FTP Server for NW 3.1x, 4.xx  (v1.10), (c) 1994 HellSoft.
Name (pandora.anglistik.uni-mainz.de:cyp): 
331 Password required.
230-User Logged.
230 Home Directory: /SYS/USER/FTP
ftp> ls
200 Command Accepted.
150 Opening ASCII data connection.
d[RWCEMFA]  1 cyp           512 Jul 01 11:49 htdocs
d[RWCEMFA]  1 cyp           512 Jul 02 08:56 config
d[RWCEMFA]  1 cyp           512 Jun 20 11:23 other
d[RWCEMFA]  1 cyp           512 Jun 20 11:23 mail
d[RWCEMFA]  1 cyp           512 Jun 20 14:35 news
d[RWCEMFA]  1 cyp           512 Jun 30 08:48 download
-[RWCEMFA]  1 *unknown    13312 May 27  1997 19calit.doc
-[RWCEMFA]  1 *unknown      237 Feb 21  1997 forfile.bat
-[RWCEMFA]  1 *unknown     1401 Mar 03  1997 intertex.txt
d[RWCEMFA]  1 cyp           512 May 15 08:55 d1
d[RWCEMFA]  1 cyp           512 May 06 10:57 drivers
d[RWCEMFA]  1 cyp           512 Mar 28 12:05 --moved-
226 Transfer complete.
ftp> close
221 Goodbye.
ftp>