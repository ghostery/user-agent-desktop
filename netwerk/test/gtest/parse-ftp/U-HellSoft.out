<!-- This Source Code Form is subject to the terms of the Mozilla Public
   - License, v. 2.0. If a copy of the MPL was not distributed with this
   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->

07-01-2002  11:49:00       <DIR>          htdocs
07-02-2002  08:56:00       <DIR>          config
06-20-2002  11:23:00       <DIR>          other
06-20-2002  11:23:00       <DIR>          mail
06-20-2002  14:35:00       <DIR>          news
06-30-2002  08:48:00       <DIR>          download
05-27-1997  00:00:00                13312 19calit.doc
02-21-1997  00:00:00                  237 forfile.bat
03-03-1997  00:00:00                 1401 intertex.txt
05-15-2002  08:55:00       <DIR>          d1
05-06-2002  10:57:00       <DIR>          drivers
03-28-2002  12:05:00       <DIR>          --moved-
